TESTS/mbedmicro-rtos-mbed/mail/main.cpp

```diff
@@ -54,8 +37,7 @@ void send_thread () {
 int main (void) {
     GREENTEA_SETUP(20, "default_auto");
 
-    Thread thread(osPriorityNormal, STACK_SIZE);
-    thread.start(send_thread);
+    Thread thread(send_thread, NULL, osPriorityNormal, STACK_SIZE);
```

Comment:
It is... I'm sure it's some merge weirdness. I'll revert that back to what it was.